### PR TITLE
Make helm parameters configurable

### PR DESCRIPTION
**Pull request by baderbuddy**
Adds options to make helm parameters configurable. I think this should be all that's needed.

### CODE REVIEW

**Review by sudermanjr — CHANGES_REQUESTED** [review]
Can you please find somewhere in the helm deploy docs to mention these settings and how they work?

**Review by sudermanjr — CHANGES_REQUESTED** [review]
Another request: I think we should probably make tis the entire "helm value" and then set the default to what the old values were.

So more like `HELM_IMAGE_TAG_VALUE_REF` would default to `image.tag` and is overridable.

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@ test/lib/
 test/pip-selfcheck.json
 node_modules
 /dist
+*.swp
```
Comment (lucasreed) [inline .gitignore:12]: This feels like something that should go into your global gitignore for your workstation.

**Review by sudermanjr — APPROVED** [review]
🎉

**Review by lucasreed — COMMENTED** [review]
One in-line comment, curious what others think.

**Pull request merged**